src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -289,12 +300,37 @@ public void checkTenantsAndConsortiumExistsOrThrow(UUID consortiumId, List<Strin
     }
   }
 
-  private void checkCentralTenantExistsOrThrow() {
-    if (tenantRepository.existsByIsCentralTrue()) {
+  private void validateConsortiumAndTenant(UUID consortiumId, Tenant tenantDto) {
+    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
+    if (tenantDto.getIsCentral() && tenantRepository.existsByIsCentralTrue()) {
       throw new ResourceAlreadyExistException("isCentral", "true");
     }
   }
 
+  private void validateCodeAndNameUniqueness(Tenant tenant) {
+    if (tenantRepository.existsByName(tenant.getName())) {
+      throw new ResourceAlreadyExistException("name", tenant.getName());
+    }
+    if (tenantRepository.existsByCode(tenant.getCode())) {
+      throw new ResourceAlreadyExistException("code", tenant.getCode());
+    }
+  }
+
+  private void validateExistTenant(TenantEntity tenant) {
+    if (Boolean.FALSE.equals(tenant.getIsDeleted()) || tenant.getIsDeleted() == null) {
```

Comment:
basically lets write migration script to update all existing tenants with isDeleted=false, it will simplify our java code to avoid additional null checks

Reply:
Hi, Regarding MODCON-130 pr https://github.com/folio-org/mod-consortia/pull/140
We agreed to write migration script to fill existing tenant tables. I checked that, there is no any migration script need because default value will be set . Also, we didn't write migration script for is_central field

![image](https://github.com/folio-org/mod-consortia/assets/113523904/0818077b-9346-496a-9f48-6ae773b41b75)

I have checked that defaultValue is being set after updating module

<img width="685" alt="Screenshot_4" src="https://github.com/folio-org/mod-consortia/assets/113523904/fae44a14-8e3c-4549-ac35-d9da9ff8568d">

<img width="670" alt="Screenshot_5" src="https://github.com/folio-org/mod-consortia/assets/113523904/1dcbe2b1-a0cc-4f66-89f8-8b201ef97936">


Is it right? Could please you guys verify that?